en/flying/fixed_wing_takeoff.md

```diff
@@ -0,0 +1,56 @@
+# Fixed Wing Takeoff
+
+PX4 supports fixed-wing takeoff in [missions](#mission) and using the [Takeoff](#takeoff) flight mode.
+Vehicles can be *catapult/hand-launched* or use the runway takeoff mode. <!-- runway support in missions? -->
+In all cases the vehicle takes off at a predefined pitch in its current direction (RC stick input is ignored).
+
+The sections below explain the main methods.
+
+
+## Takeoff Flight Mode {#takeoff}
+
+[Takeoff Mode](../flight_modes/takeoff.md#fixed_wing) enables takeoff using either *catapult/hand-launch* (default) or *runway takeoff*.
+
+### Catapult/Hand-Launch Mode
+
+When armed and in takeoff mode the vehicle waits to detect launch (i.e. from acceleration due to catapult or hand launch). 
+On launch detection, the vehicle ramps up to to maximum throttle ([RWTO_MAX_THR](#RWTO_MAX_THR)) in about 2 seconds and then performs a full throttle climbout at a 10 degree pitch.
+The vehicle climbout phase ends once the vehicle reaches the correct altitude (defined by [FW_CLMBOUT_DIFF](#FW_CLMBOUT_DIFF)), after which regular navigation proceeds.
+All RC stick movement is ignored until climbout ends.
+
+> **Warning** The default climbout pitch may not be suitable for some vehicles. <!-- see https://github.com/PX4/Firmware/pull/9243 -->
+
+<span></span>
+> **Tip** The vehicle must be flat and level on release (it is important the nose is neither up or down, and that the vehicle is not rolling/yawing).
+
+To launch in this mode:
+- Arm the vehicle.
+- Put the vehicle into *Takeoff mode*.
+- Launch/throw the vehicle (firmly) directly into the wind.
+  For hand launch ensure that the vehicle is level.
+  
+
+> **Tip** Some users report improved takeoff performance by pre-triggering launch detection and ramping up to full throttle before release (to trigger launch detection pump/shake the aircraft forward).
+
+
+### Runway Takeoff Mode
+
```

Comment:
FYI, I just link to the flight mode docs for this case, because fully documented there.